cmd/adapter/api/api.go

```diff
@@ -357,6 +365,23 @@ func toGravityLogEntries(evs []*api.LogEvent) ([]string, error) {
 	return entries, nil
 }
 
+func parseCSVIntoMap(csv string) (results map[string]string) {
+
+	// iterate over the elements in the form `elem1, elem2, elem3, ...`
+	for _, elem := range strings.Split(csv, ",") {
+
+		// each element should be in the form `key=value`
+		elemSplit := strings.Split(elem, "=")
```

Comment:
Can you describe the input format in more detail - maybe as a Godoc for the method?
Can a value be a string with whitespace and potentially include an embedded equals sign if quoted?

Reply:
explanation comment and FIXME about the '=' char added.